data/json/item_groups.json

```diff
@@ -3286,30 +3286,42 @@
       ["survnote", 2]
     ]
   },{
-      "type" : "item_group",
-    "id" : "mansion_books",
-    "//" : "Those rich mansion people have all sorts of weird books.",
-    "items":[
-      ["textbook_weapwest", 5],
-      ["textbook_weapeast", 5],
-      ["textbook_armwest", 5],
-      ["textbook_armeast", 5],
-      ["textbook_armschina", 5],
-      ["textbook_gaswarfare", 5],
-      ["photo_album", 10],
-      ["fun_survival", 10],
-      ["brewing_cookbook", 8],
-      ["textbook_robots", 5],
-      ["manual_dodge", 5],
-      ["novel_experimental", 8],
-      ["novel_swash", 10],
-      ["manual_fencing", 3],
-      ["manual_niten", 3],
-      ["novel_samurai", 10],
-      ["plays_book", 10],
-      ["poetry_book", 10],
-      ["carpentry_book", 8],
-      ["scots_cookbook", 10]
+    "id": "mansion_books",
+    "type": "item_group",
+    "//": "Those rich mansion people have all sorts of weird books.",
+    "items": [
+      [ "textbook_weapwest", 15 ],
+      [ "textbook_weapeast", 15 ],
+      [ "textbook_armwest", 15 ],
+      [ "textbook_armeast", 15 ],
+      [ "textbook_armschina", 15 ],
+      [ "textbook_gaswarfare", 15 ],
+      [ "photo_album", 30 ],
+      [ "fun_survival", 30 ],
+      [ "brewing_cookbook", 24 ],
+      [ "textbook_robots", 15 ],
+      [ "manual_dodge", 15 ],
+      [ "novel_experimental", 24 ],
+      [ "novel_swash", 30 ],
+      [ "manual_fencing", 9 ],
+      [ "manual_niten", 9 ],
+      [ "novel_samurai", 30 ],
+      [ "plays_book", 30 ],
+      [ "poetry_book", 30 ],
+      [ "carpentry_book", 24 ],
+      [ "scots_cookbook", 30 ],
```

Comment:
Ooh my bad. I didn't notice the numbers changed for the other entries. That makes sense then. I would recommend that the "id" and "type" lines switch place to match the format of the other entries.